static-site/content/blog/2024-06-18-h5n1-cattle-outbreak-analysis-and-resources.md

```diff
@@ -0,0 +1,132 @@
+---
+author: "Trevor Bedford, Louise Moncla, Jover Lee, James Hadfield, John Huddleston"
+date: "2024-06-18"
+title: "Phylogenetic analysis of H5N1 cattle outbreak and curated genomic data"
+---
+**Trevor Bedford**, **Louise Moncla**, **Jover Lee**, **James Hadfield**, **John Huddleston**
+
+# Phylogenetic analysis
+
+The 2.3.4.4b clade of influenza subtype H5N1 has been remarkably successful in its global spread with arrival into Europe in mid-2020, into North America in late 2021 and into South America in mid-2022 (**Fig. 1**).
+Although sustained transmission and spread has been largely within birds, these viruses have infected a number of species of non-human mammals, including [foxes, skunks and domestic cats](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=species_group&f_species_group=Mammal-%20Terrestrial&r=division&transmissions=hide) which appear to be dead-end hosts to infection, but also [seals in North America](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=species_group&f_species_group=Mammal-%20Marine&r=division&transmissions=hide) and [sea lions in South America](https://www.nature.com/articles/s41467-023-41182-0), where sustained transmission outside of birds is highly likely.
+
+[![fig1](img/h5n1_cattle_outbreak_fig1_ha_all_time_region.png)](https://nextstrain.org/avian-flu/h5n1/ha/all-time@2024-06-18)
+**Figure 1. Broader context of H5N1 evolution and spread.**
+Available as [nextstrain.org/avian-flu/h5n1/ha/all-time](https://nextstrain.org/avian-flu/h5n1/ha/all-time@2024-06-18).
+
+More recently, this clade of H5N1 was discovered circulating in dairy cows in the US with cases [first reported in Texas in March 25, 2024](https://www.aphis.usda.gov/news/agency-announcements/federal-state-veterinary-public-health-agencies-share-update-hpai).
+Consensus genomes from infected dairy cattle have been shared to GenBank primarily by the [National Veterinary Services Laboratories (NVSL)](https://www.aphis.usda.gov/labs/about-nvsl) of the [Animal and Plant Health Inspection Service (APHIS)](https://www.aphis.usda.gov/) of the U.S. Department of Agriculture (USDA).
+We find, in accordance to results by [Nguyen et al](https://www.biorxiv.org/content/10.1101/2024.05.01.591751v1) and by [Worobey et al](https://virological.org/t/preliminary-report-on-genomic-epidemiology-of-the-2024-h5n1-influenza-a-virus-outbreak-in-u-s-cattle-part-1-of-2/970), that cattle-associated viruses form a distinct monophyletic clade across all 8 segments (**Fig. 2**).
+Comparing phylogenetic structure across segments shows reassortment in birds prior to cattle spillover, but that the [cattle clade appears be non-reassorting](https://nextstrain.org/avian-flu/h5n1/ha/2y@2024-06-18:avian-flu/h5n1/pb2/2y@2024-06-18?c=host&m=div).
```

Comment:
minor edit: "cattle clade appears *to* be"

Reply:
Thanks for the catch. Resolved in de14773b68f03ec6c5379397e817e7d37a009cc9.